src/Applications/LDAS_App/ldas_setup

```diff
@@ -671,6 +671,14 @@ class LDASsetup:
         if 'SURFLAY' in self.rqdExeInp :
            dzsf = self.rqdExeInp['SURFLAY']
 
+        wemin_in  = '13'  #  WEmin input/output for scale_catch(cn),
```

Comment:
This comment does not make it clear that these are "dummy" inputs, and that the resulting restarts will require long-term spin-up.

Reply:
in ldas_setup and add
print'a brief description'
print '#WEMIN_IN:'
print '#WEMIN_OUT:'

right after below text (then it will appear in sample.exec)
print
print 'RESTART :'
print '#RESTART_ID :'
print '#RESTART_PATH :'
print '#RESTART_DOMAIN :'

Reply:
as discussed offline, we'll just beef up the comment. WEMIN's won't be inputs in exec.